CHANGELOG.unreleased.md

```diff
@@ -9,6 +9,7 @@
 
 - [Remotes] Prevent remote path from ending with `xo-vm-backups` as it's usually a mistake
 - [OVA export] Speed up OVA generation by 2. Generated file will be bigger (as big as uncompressed XVA) (PR [#6487](https://github.com/vatesfr/xen-orchestra/pull/6487))
+- [Proxy] Ability to connect to an existing proxy from the `Proxies` view (PR [#6556](https://github.com/vatesfr/xen-orchestra/pull/6556))
```

Comment:
Why did you chose the term *connect* instead of *register*?

*register* is the term used in XO API, which is why it was used in the task description that led to this PR.

Reply:
I used "connect" because for me it corresponds more with what the feature offers.
I will rename with `register` so.

Reply:
I don't have a strong opinion on this but I think having different names in the UI and the API is not a good idea.